assignment6.py

```diff
@@ -0,0 +1,23 @@
+def swap(l, begin, end):
+    tmp = l[begin]
+	l[begin] = l[end]
+	l[end] = tmp
+
+def rearrage(src, target):
+
+	free = 0
+
+	src_ind = dict([(a, i) for i, a in enumerate(src)])
```

Comment:
I like the idea of the dictionary! "a" is the variable of the car number and "i" is the index, right?